assets/src/blocks/HubspotForm/hooks/useToAttribute.js

```diff
@@ -0,0 +1,23 @@
+export const useToAttribute = (setAttributes) => {
+  const toAttribute = (attributeName, maxLength = -1) => value => {
+    if(setAttributes) {
+      if(maxLength > -1) {
+        if(value.length < maxLength) {
+          setAttributes({
+            [attributeName]: value,
+          });
+        } else {
+          setAttributes({
+            [attributeName]: value.slice(0, maxLength),
```

Comment:
We probably don't want to enforce it this way.

What if someone copy pastes text into the block from somewhere? Also if you would start changing a word in the middle of your sentence, it will disappear at the end and won't come back if you delete the extra characters again.

If the text is too long, we should show them a warning, and optionally block the post from saving until they fixed it. We can do this using validation on attributes, [like for slides on the Carousel Header block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/9dd382fc04caa03d56f62b4038e21f9dac3d3cd1/assets/src/blocks/CarouselHeader/CarouselHeaderBlock.js#L26-L35).

But it's also something we can do in a follow up. Definitely should be fixed before using the block in production.